index.html

```diff
@@ -0,0 +1,187 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio</title>
+  <link href="https://fonts.googleapis.com/css2?family=Inter&display=swap" rel="stylesheet">
+
+  <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+  <!-- Navbar -->
+
+  <header>
+    <div class="nav-container">
+      <nav class="nav-bar">
+        <div class="logo">
+          <div class="logo-placeholder">
+            <a href="#">
+              <h1>Welcome</h1>
+            </a>
+          </div>
+        </div>
+        <div class="harmburger">
+          <img src="./images/Union.svg" alt="Toggle button to display hidden nav items">
+        </div>
```

Comment:
- [ ] Kindly wrap the hamburger button inside a `nav` tag.

Reply:
The union image is wrapped inside the nav tag.